src/Screens/Flows/App/AppsScreen/Components/FavoritesBottomSheet.tsx

```diff
@@ -167,9 +171,13 @@ export const FavoritesBottomSheet = React.forwardRef<BottomSheetModalMethods, Pr
                         windowSize={5}
                         ListEmptyComponent={
                             <ListEmptyResults
-                                subtitle={LL.FAVOURITES_DAPPS_NO_RECORDS()}
-                                icon={"icon-search"}
+                                subtitle={LL.FAVOURITES_DAPPS_EMPTY_LIST()}
+                                icon={"icon-alert-circle"}
+                                iconColor={theme.colors.actionBottomSheet.emptyFavoritesIcon.color}
                                 testID="empty-results"
+                                minHeight={emptyStateMinHeight}
+                                iconStyle={styles.emptyIcon}
+                                subtitleColor={theme.colors.actionBottomSheet.text}
```

Comment:
On Figma, in light mode, the color is Gray/600, I think this is 700 on our config

Reply:
Addressed